### ISSUE TITLE
dev: refactor environment variables, and do environment validation before RPC start

### ISSUE THREAD

**Issue by bajpai244** [kkrt-labs/kakarot-rpc#199]
tldr:
- before startup of rpc make sure essential environment variables are there and not empty { will act as a great smoke test }
- group parsing of all these variables in a single file, where they are exposed as constants, but their value is provided at runtime ( might be prefixed with `ENV` to specify these are environment variables and not normal constants ).

This will improve the readability and quality of the code, as we would be doing environment validation, and variable parsing only once, and can then use this variable like how we use constants. We can make use of lazy_static! strategy.

**Comment by bajpai244** [comment]
working on this one!

@Eikix you can assign me for this.
